README.md

```diff
@@ -14,6 +14,14 @@ This repository shows how to build a Machine Learning Pipeline for [Semantic Seg
 - [ ] Modify [configs.py](https://github.com/deep-diver/segformer-in-ml-pipeline/blob/main/training_pipeline/pipeline/configs.py) to reflect changes.
 - [ ] (Optional) Add custom TFX component to dynamically inject hyperameters to search with `Tuner`.
 
+## Formatting
+
+We use [black](https://pypi.org/project/black/) to lint Python source files. In order to apply it effortlessly, it is recommended to use [pre-commit](https://pre-commit.com/) hook, so run the following command before any contributions.
+
+```shell
+$ pre-commit install --hook-type pre-push
+```
```

Comment:
I already have my VS Code configured such that for every CMD + S it will run formatting.

Reply:
Just to unify the formatting style. Are you using black?

Reply:
I am using black, yes. This is how my configuration looks like: 

```json
    "editor.formatOnSave": true,
    "diffEditor.ignoreTrimWhitespace": false,
    "python.formatting.blackPath": "/Users/sayakpaul/.local/bin/.virtualenvs/formatting/bin/black",
    "python.formatting.blackArgs": [
        "--line-length=80"
    ],
    "python.sortImports.path": "/Users/sayakpaul/.local/bin/.virtualenvs/plm/bin/isort",
    "editor.codeActionsOnSave": {
        "source.organizeImports": true
    },
    "python.sortImports.args": [
        "--profile",
        "black",
        "--line-length",
        "80"
    ],
```